src/Workspaces/Remote/ServiceHub/Host/SolutionAssetCache.cs

```diff
@@ -138,42 +115,10 @@ private async Task CleanAssetsAsync()
         while (!cancellationToken.IsCancellationRequested)
         {
             await CleanAssetsWorkerAsync(cancellationToken).ConfigureAwait(false);
-
-            ForceGC();
```

Comment:
>ForceGC(); [](http://example.com/codeflow?start=12&length=10)

I can do this in a separate PR, but it kinda bugs me that this has it's own dedicated thread for this. ABWQ usage would prevent the need for that, right?

Reply:
Never mind, expanded my search and there are other activities on this thread